### PR TITLE
Checksum Verification Scheduler

### PR DESCRIPTION
**Pull request by slint**

### CODE REVIEW

File: invenio_files_rest/tasks.py [hunk 1]
```diff
@@ -50,16 +55,109 @@ def progress_updater(size, total):
...
+    files = files.yield_per(1000)
+    scheduled_file_ids = []
+    total_size = 0
+    for f in files:
+        # Add at least the first file, since it might be larger than "max_size"
+        scheduled_file_ids.append(str(f.id))
+        total_size += f.size
+        if max_size and max_size < total_size:
+            break
+    print('Sending this files for checksum:', scheduled_file_ids)
```
Comment (lnielsen) [inline invenio_files_rest/tasks.py]: Is the print supposed to be here? If yes, then perhaps better to use the task logger.

Comment (slint) [inline invenio_files_rest/tasks.py]: Slipped during debug. Removed and pushed some tests to pass the coverage as well

**Comment by lnielsen** [comment]
Test passes on lowest requirements, so I think it's an Celery v3 vs v4 issue.

**Comment by slint** [comment]
@lnielsen Everything passing now. There's no easy way to test for warnings being logged though...

**Pull request merged**